wagtail/admin/templates/wagtailadmin/tables/download_cell.html

```diff
@@ -1 +1 @@
-<td><a href="{{ download_url }}" class="nolink" download>{{ value }}</a></td>
+<td class="wrap-anywhere"><a href="{{ download_url }}" class="nolink" download>{{ value }}</a></td>
```

Comment:
We have Tailwind configured with the `w-` prefix, so this should work (after running `npm run build`):

```suggestion
<td class="w-wrap-anywhere"><a href="{{ download_url }}" class="nolink" download>{{ value }}</a></td>
```